schemas/User.yaml

```diff
@@ -11,4 +12,42 @@ properties:
   password:
     description: The secure password of the user. must be greater than 8 characters long
     type: string
-    
\ No newline at end of file
+  email:
+    description: the email address of the user.
+    type: string
+    example: test@test.com
+  dateOfBirth:
+    description: user's date of birth
+    type: date
+    example: 17/02/2003
+  phone:
+    description: phone number of the user, use for verifiication.
+    type: string
+    example: ysf684sgd76g8r
+  profilePicture:
+    description: user's image
+    type: object
```

Comment:
pictures will be stored in an external object store and only the URL will be kept in our system. so this should be a string property representing the picture url